draft-ietf-quic-transport.md

```diff
@@ -2352,17 +2337,56 @@ Application Error Code:
   {{app-error-codes}}).
 
 
+## PATH_CHALLENGE Frame {#frame-path-challenge}
 
-## PONG Frame {#frame-pong}
+Endpoints can use PATH_CHALLENGE frames (type=0x0e) to check reachability to the
+peer, to verify a new path's PMTU, and for address validation during connection
+establishment and connection migration.
 
-The PONG frame (type=0x0d) is sent in response to a PING frame that contains
-data.  Its format is identical to the PING frame ({{frame-ping}}).
+PATH_CHALLENGE frames contain a variable-length payload.
```

Comment:
A fixed size is better.  No chance of messing it up that way.  As I said above, I think that 12 octets is pretty good (that's the size of the STUN transaction ID).

Reply:
+1 for fixed size.